smart_control/utils/environment_utils.py

```diff
@@ -56,25 +41,3 @@ def set_action_normalization_constants(
       min_normalized_value,
       max_normalized_value,
   )
-
-
-@gin.configurable
-def get_zones_from_config(
-    configuration_path: str,
-) -> Sequence[smart_control_building_pb2.ZoneInfo]:
-  """Loads up the zones as a gin macro."""
-  with gin.unlock_config():
-    reader = reader_lib_google.RecordIoReader(input_dir=configuration_path)
-    zone_infos = reader.read_zone_infos()
-    return zone_infos
-
-
-@gin.configurable
-def get_devices_from_config(
```

Comment:
This `get_devices_from_config` function is unused anywhere in the codebase. Removing it also fixes an undefined variable reference error, which was the original goal.